armi/nuclearDataIO/cccc/__init__.py

```diff
@@ -58,7 +58,7 @@
 
 .. [CCCC-IV] R. Douglas O'Dell, "Standard Interface Files and Procedures for Reactor Physics
              Codes, Version IV," LA-6941-MS, Los Alamos National Laboratory (September 1977).
-             Web. doi:10.2172/5369298. (`OSTI <https://www.osti.gov/biblio/5369298>`_)
+             Web. doi:10.2172/5369298. (`OSTI <https://www.osti.gov/biblio/5369298>`__)
```

Comment:
Can you explain this one to me?

I just don't get the change, that's all.

The link is currently working on the docs:  https://terrapower.github.io/armi/.apidocs/armi.nuclearDataIO.cccc.html#module-armi.nuclearDataIO.cccc

Reply:
This is switching the link to be "anonymous", so the `OSTI` label isn't an explicit target. Without this, `OSTI` is created as a reference target twice, throwing a `WARNING: Duplicate explicit target name`. See this for a detailed explanation: https://stackoverflow.com/questions/74252545/differences-between-anonymous-and-non-anonymous-hyperlinks-in-sphinx